app/[locale]/about/page.tsx

```
@@ -0,0 +1,206 @@
+                </h3>
+                <p className="text-sm text-slate-600 dark:text-slate-400">
+                  Learn how we protect your data
+                </p>
+              </div>
+              <svg
+                className="w-5 h-5 text-slate-400 group-hover:text-theme-primary-600 dark:group-hover:text-theme-primary-400 group-hover:translate-x-1 transition-all duration-300"
+                fill="none"
+                stroke="currentColor"
+                viewBox="0 0 24 24"
+              >
+                <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)